examples/Nonnominal_PSF_simulations_use_examples.ipynb

```diff
@@ -0,0 +1,862 @@
+    "                                                                yam.info['v3'], \n",
+    "                                                                yam.info['yamlfile'])])\n",
+    "\n",
+    "print('Example files for each pointing:')\n",
+    "print('--------------------------------')\n",
+    "plotted_points = []\n",
+    "for i_point, (v2, v3, filename) in enumerate(all_pointings):\n",
+    "    if (v2, v3) not in plotted_points:\n",
+    "        plotted_points.append((v2, v3))\n",
+    "        plt.scatter(v2, v3, marker='*', s=500, \n",
+    "                    label='Pointing {}/{}'.format(i_point + 1, len(all_pointings)))\n",
+    "        print('{}. {}'.format(i_point + 1, filename))\n",
+    "\n",
+    "plt.legend()\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is easy for 1153 Obs. 1, since all 18 exposures have the same pointing. We'll just choose a YAML that has a detector and filter that matches the library files we have created so far - NRCA3 and F212N.\n",
+    "\n",
+    "Let's choose `jw01153001001_01101_00001_nrca3.yaml`.\n",
+    "\n",
+    "*See [JDox](https://jwst-docs.stsci.edu/display/JDAT/File+Naming+Conventions+and+Data+Products) for a detailed explanation of the MIRaGE YAML file name format.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select one YAML where the target will be in the field of view\n",
+    "yfiles = glob(os.path.join(yaml_dir, '*.yaml'))\n",
+    "file_name_to_match = 'jw01153001001_01101_00001_nrca3'\n",
+    "yaml_to_sim = os.path.abspath([y for y in yfiles if file_name_to_match in y][0])\n",
+    "print(yaml_to_sim)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "<a id='simulate_images'></a>\n",
+    "# 4. Simulate image with MIRaGe\n",
+    "\n",
+    "Finally, we can run MIRaGe to generate a seed image simulation of our unstacked mirror state during OTE-17.\n",
+    "\n",
+    "From here on out, from the user perspective, the simulation process is identical to that of nominal imaging cases (see the [imaging example notebook](#Imaging_simulator_use_examples.ipynb). To reiterate, it is the specifications made in the YAML files that enable the simulation of nonnominal mirror simulations with MIRaGe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# Run the image simulator using the input defined in yaml_to_sim\n",
+    "img_sim = imaging_simulator.ImgSim()\n",
+    "img_sim.paramfile = yaml_to_sim\n",
+    "img_sim.create()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot the seed image, dark image, and final exposure simulation\n",
```

Comment:
I think it's the negative pixels that were showing up white and making the image look noisy. I adjusted the plots so all negative pixels are just set to 0.1, and also adjusted the scale range, I think that helped a bit.

Reply:
For log scales it is useful to do e.g.
```
cmap = whichever_color_map_you_want_to_use
cmap.set_bad(cmap(0))
plt.imshow(stuff, cmap=cmap)
```

This changes the behavior such that instead of zero or negative pixels showing up as white, they show up as the same color as whatever the bottom of the color map is. This is often the more desired visual behavior, in my opinion.